packages/slice-machine/src/components/SideNav/SideNav.css.ts

```diff
@@ -34,15 +34,10 @@ export const root = style([
     backgroundColor: colors.grey2,
     display: "flex",
     flexDirection: "column",
-    height: "100%",
-    minHeight: "100vh",
-    paddingBottom: 32,
```

Comment:
⚠ Why removing this padding-bottom here to after add a margin-bottom to last list list `128`?

Reply:
Ok I see :/